.chloggen/split-emf-log-when-buckets-larger-than-100.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awsemfexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Split EMF log to multiple log splits when buckets larger than 100.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [242]
```

Comment:
This points to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/242. I suppose you had something else in mind?

Reply:
Sorry, I was meant to use PR number. Updated.